moped-editor/src/layouts/DashboardLayout/TopBar.js

```diff
@@ -1,16 +1,13 @@
 import React, { useState } from "react";
 import { Link as RouterLink } from "react-router-dom";
-import clsx from "clsx";
-import PropTypes from "prop-types";
-import { AppBar, Box, Hidden, Toolbar, Tabs, Tab } from "@mui/material";
-import makeStyles from "@mui/styles/makeStyles";
-import { Alert } from "@mui/material";
+import { AppBar, Box, Hidden, Toolbar, Tabs, Tab, Alert } from "@mui/material";
 import Logo from "src/components/Logo";
 import { CanAddProjectButton } from "src/views/projects/projectsListView/ProjectListViewCustomComponents";
 import MobileDropdownMenu from "src/layouts/DashboardLayout/NavBar/MobileDropdownMenu";
 import DropdownMenu from "src/layouts/DashboardLayout/NavBar/DropdownMenu";
 import NavigationSearchInput from "src/layouts/DashboardLayout/NavBar/NavigationSearchInput";
-import NavLink from "src/components/NavLink";
+import { NavLink } from "react-router-dom";
```

Comment:
I believe we will also be able to delete this custom component that was made to support some deprecated behavior in `react-router-dom` and be able to just use the standard component. The custom component is currently used in a dozen or so files.

Reply:
you're right, thanks for pointing that out! glad we can delete this custom component :)